.chglog/config.yml

```diff
@@ -0,0 +1,12 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: Changelog
+  repository_url: https://github.com/spinnaker/kustomization-base
+options:
+  header:
+    pattern: "^(\\w*)(?:\\(([\\w\\$\\.\\-\\*\\s]*)\\))?\\:\\s(.*)$"
```

Comment:
Want to add a summary of your comment from the PR description here as documentation, in case anyone less familiar with our commit conventions is trying to understand this regex?

```
# Only include commits whose message comply with the convention <type>(<scope>): <subject>
```

Reply:
Added: [`ab67137` (#27)](https://github.com/spinnaker/kustomization-base/pull/27/commits/ab67137c639181544ad68a00f26da0cdd5413cc4)